app/views/layouts/_title.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= t(:app_name) %>
+<% if current_page?(new_user_session_path) ||
```

Comment:
A follow-up to this PR is to add <%title %>s to all of these devise views so that we can get rid of all of this logic